src/sentry/incidents/events.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+from sentry import analytics
+
+
+class BaseIncidentEvent(analytics.Event):
+    attributes = (
+        analytics.Attribute('incident_id'),
+        analytics.Attribute('organization_id'),
+        analytics.Attribute('incident_type'),
+    )
+
+
+class IncidentCreatedEvent(BaseIncidentEvent):
+    type = 'incident.created'
```

Comment:
FYI these need to be added here: https://github.com/getsentry/reload/blob/master/reload_app/events.py

Reply:
@billyvg Ahh will do. Does this need to happen before landing this pr?

Reply:
In general it should but since this is an internal feature atm it won't matter really